### ISSUE TITLE
[1.0.0-rc1] 'docker volume ls' output not showing node name (swarm 1.0.0-rc1 and engine 1.9.0-rc1)

### ISSUE THREAD

**Issue by mbentley** [docker/swarm#1302]
Description of problem:
'docker volume ls' output not showing nodes when getting info via swarm 1.0.0-rc1.  This makes it difficult to manage the local volumes that are running across a swarm cluster since you can't be sure which node the local volumes exist when they do not exist across all nodes.

`docker version`:

```
$ docker-1.9.0-rc1 version
Client:
 Version:      1.9.0-rc1
 API version:  1.21
 Go version:   go1.4.3
 Git commit:   9291a0e
 Built:        Wed Oct 14 03:29:07 UTC 2015
 OS/Arch:      darwin/amd64

Server:
 Version:      swarm/1.0.0
 API version:  1.16
 Go version:   go1.5.1
 Git commit:   fe88189
 Built:
 OS/Arch:      linux/amd64
```

`docker info`:

```
$ docker-1.9.0-rc1 info
Containers: 11
Images: 18
Engine Version:
Role: primary
Strategy: spread
Filters: affinity, health, constraint, port, dependency
Nodes: 3
 orca2.mac: 192.168.99.102:12376
  └ Containers: 2
  └ Reserved CPUs: 0 / 1
  └ Reserved Memory: 0 B / 1.021 GiB
  └ Labels: executiondriver=native-0.2, kernelversion=4.1.10-boot2docker, operatingsystem=Boot2Docker 1.9.0-rc1 (TCL 6.4); master : 4187d2c - Wed Oct 14 16:48:27 UTC 2015, provider=virtualbox, storagedriver=aufs
 orca3.mac: 192.168.99.103:12376
  └ Containers: 2
  └ Reserved CPUs: 0 / 1
  └ Reserved Memory: 0 B / 1.021 GiB
  └ Labels: executiondriver=native-0.2, kernelversion=4.1.10-boot2docker, operatingsystem=Boot2Docker 1.9.0-rc1 (TCL 6.4); master : 4187d2c - Wed Oct 14 16:48:27 UTC 2015, provider=virtualbox, storagedriver=aufs
 orca.mac: 192.168.99.101:12376
  └ Containers: 7
  └ Reserved CPUs: 0 / 1
  └ Reserved Memory: 0 B / 1.021 GiB
  └ Labels: executiondriver=native-0.2, kernelversion=4.1.10-boot2docker, operatingsystem=Boot2Docker 1.9.0-rc1 (TCL 6.4); master : 4187d2c - Wed Oct 14 16:48:27 UTC 2015, provider=virtualbox, storagedriver=aufs
CPUs: 3
Total Memory: 3.064 GiB
Name: e2755041b163
```

`uname -a`:

```
$ uname -a
Darwin wafflemaker 15.0.0 Darwin Kernel Version 15.0.0: Wed Aug 26 16:57:32 PDT 2015; root:xnu-3247.1.106~1/RELEASE_X86_64 x86_64
```

Environment details (AWS, VirtualBox, physical, etc.):
3 node swarm 1.0.0-rc1 cluster; each engine running 1.9.0-rc1; OS X client running 1.9.0-rc1

How reproducible:
Every time with the RC of each.

Steps to Reproduce:
1. Create a swarm cluster using swarm-1.0.0-rc1 and docker engine 1.9.0-rc1
2. Point your docker client to the swarm master
3. Run a `docker volume ls`

Actual Results:

```
$ docker-1.9.0-rc1 volume ls
DRIVER              VOLUME NAME
local               orca-config
local               c14a190cf7894766a445a680e8d35a5ac83ee695cbbee217163e662e1150a91c
local               orca-root-ca
local               orca-swarm-root-ca
local               orca-kv
local               c88e28c29ccbab06094ad21dfc06bc71a52c3f47f139e26c122e5ea0a0bb0f01
local               orca-server-certs
local               orca-swarm-node-certs
local               orca-swarm-kv-certs
local               ae5a6d904f232e093008cc8b4ee2cd309308afa995cc5d25f1773eb46d75735b
local               d7aa9b8e0961024e917eb444c54ab30a11c358962032785ded28c6fb1d7b8a77
local               orca-swarm-node-certs
local               937e5d32a838d976e02fba8fa4664115022f9651f46f927c1f016de52a4d8705
local               orca-swarm-node-certs
local               e2d37a7a33b64d569932a10c0751714eee33e4043c53d131f93194349bb02923
```

Expected Results:
Similar output but with node names added to the volume names

Additional info:
Is this actually expected behavior?  I just noticed that when I create a volume via swarm, it creates it everywhere.

**Comment by aluzzardi** [comment]
ping @vieux

**Comment by vieux** [comment]
@mbentley sorry I didn't reply earlier, we refactored the UX in the rc2 and have a few fix in master. We are going to release rc3 soon.

**Comment by thaJeztah** [comment]
Is this fixed in the current release?

**Comment by mbentley** [comment]
It doesn't appear so:

```
$ docker version
Client:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.3
 Git commit:   a34a1d5
 Built:        Fri Nov 20 17:56:04 UTC 2015
 OS/Arch:      darwin/amd64

Server:
 Version:      swarm/1.0.0
 API version:  1.21
 Go version:   go1.5.1
 Git commit:   087e245
 Built:
 OS/Arch:      linux/amd64
$ docker info
Containers: 35
Images: 68
Role: primary
Strategy: spread
Filters: health, port, dependency, affinity, constraint
Nodes: 3
 orca2.mac: 192.168.56.12:12376
  └ Containers: 10
  └ Reserved CPUs: 0 / 1
  └ Reserved Memory: 0 B / 1.794 GiB
  └ Labels: executiondriver=native-0.2, kernelversion=3.16.0-51-generic, operatingsystem=Ubuntu 14.04.3 LTS, storagedriver=aufs
 orca3.mac: 192.168.56.13:12376
  └ Containers: 9
  └ Reserved CPUs: 0 / 1
  └ Reserved Memory: 0 B / 1.794 GiB
  └ Labels: executiondriver=native-0.2, kernelversion=3.16.0-51-generic, operatingsystem=Ubuntu 14.04.3 LTS, storagedriver=aufs
 orca.mac: 192.168.56.11:12376
  └ Containers: 16
  └ Reserved CPUs: 0 / 1
  └ Reserved Memory: 64 MiB / 2.053 GiB
  └ Labels: executiondriver=native-0.2, kernelversion=3.16.0-51-generic, operatingsystem=Ubuntu 14.04.3 LTS, storagedriver=aufs
CPUs: 3
Total Memory: 5.641 GiB
Name: f6949a90ce9f
$ docker volume ls
DRIVER              VOLUME NAME
local               ucp-server-certs
local               ucp-swarm-controller-certs
local               2a9aa5151ef30e504b1af383e141661a7afa6b8e176f1f32d946f1378123576e
local               b334d329d5b66e3f7898a2d18d090f1713e3fdb4aedf1808a158f1ddb6bc902a
local               ce7ee8602ffab4497f91f439e49aae4b5640b424df3d187e686cc06c88518a3d
local               ucp-swarm-root-ca
local               bitbucket-data
local               7777aafc838c6021d97b35147e6a653db68af961d0a9eb780f87bf5f072f5067
local               ucp-root-ca
local               ucp-kv
local               ucp-swarm-kv-certs
local               839e60853898c144c3acaf8600a6742846f0de4a490c9319a5b770637e9e555e
local               7e07359f7a942967d3e6a5378fe45a11b1eccb313d6bec3597b7ea3f7d256b5a
local               ucp-swarm-node-certs
local               7ca4885402efab1e62d9cdaeb098363c9c58cd999f83a27161118bee2f4d8e53
local               b4361c4640af785135d9bbd94616e47a767457f263738e21d34939bf3285aa1e
local               ucp-swarm-node-certs
local               a213ed3d0b431e0fe635e785fb839d54bebc6db9a0c9a8692ddbf2c9c6cf3e75
local               ucp-swarm-node-certs
local               33bcbee562549c4991a682367ae50341fd277f63d03c4c32800bf63ec306610a
local               915d5f04fc2ff57ff4534c1a25301251a56fa0f8317d591afa4ce3ab050c1511
```

**Comment by vieux** [comment]
@mbentley it looks like you tried with 1.0.0, can you try with 1.0.1 ?

Thanks.

**Comment by mbentley** [comment]
I'm on swarm:1.0.1 and `docker volume ls` doesn't provide which host a local volume is found on, same as above:
```
$ docker version
Client:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.3
 Git commit:   a34a1d5
 Built:        Fri Nov 20 17:56:04 UTC 2015
 OS/Arch:      darwin/amd64

Server:
 Version:      swarm/1.0.1
 API version:  1.21
 Go version:   go1.5.2
 Git commit:   744e3a3
 Built:
 OS/Arch:      linux/amd64
$ docker info
Containers: 46
Images: 94
Role: replica
Primary: 192.168.56.12:2376
Strategy: spread
Filters: health, port, dependency, affinity, constraint
Nodes: 3
 orca2.mac: 192.168.56.12:12376
  └ Status: Healthy
  └ Containers: 15
  └ Reserved CPUs: 0 / 1
  └ Reserved Memory: 0 B / 1.794 GiB
  └ Labels: executiondriver=native-0.2, kernelversion=3.13.0-74-generic, operatingsystem=Ubuntu 14.04.3 LTS, storagedriver=aufs
 orca3.mac: 192.168.56.13:12376
  └ Status: Healthy
  └ Containers: 15
  └ Reserved CPUs: 0 / 1
  └ Reserved Memory: 0 B / 1.794 GiB
  └ Labels: executiondriver=native-0.2, kernelversion=3.13.0-74-generic, operatingsystem=Ubuntu 14.04.3 LTS, storagedriver=aufs
 orca.mac: 192.168.56.11:12376
  └ Status: Healthy
  └ Containers: 16
  └ Reserved CPUs: 0 / 1
  └ Reserved Memory: 64 MiB / 2.053 GiB
  └ Labels: executiondriver=native-0.2, kernelversion=3.13.0-74-generic, operatingsystem=Ubuntu 14.04.3 LTS, storagedriver=aufs
CPUs: 3
Total Memory: 5.642 GiB
Name: f6949a90ce9f
$ docker volume ls
DRIVER              VOLUME NAME
local               ucp-server-certs
local               ucp-swarm-kv-certs
local               33bcbee562549c4991a682367ae50341fd277f63d03c4c32800bf63ec306610a
local               3e48d86a38486e2ed7a757f07f24a7480a8898828659fb6c3d4be5b3942e7770
local               7409dd00ee9dfa3b3e634717b687134c05d98e3dc8c7aa2184b13aa3f05a695b
local               32c99861b2d4762977a7c27da58e4a2a9a24e8557c521907b25e8a450bb3d95b
local               ucp-kv
local               ucp-swarm-node-certs
local               2863d81ff89e0967841e58518428e869ed564553c2ebac2f72f3e51a114b4439
local               ucp-swarm-controller-certs
local               915d5f04fc2ff57ff4534c1a25301251a56fa0f8317d591afa4ce3ab050c1511
local               postgres-data
local               7e07359f7a942967d3e6a5378fe45a11b1eccb313d6bec3597b7ea3f7d256b5a
local               ucp-server-certs
local               ucp-swarm-node-certs
local               ucp-kv
local               bitbucket-data
local               68fd34a7f1104a1198a0ac02cf0d6d22502f5f6c8ecbbd0a49b9accd3cac0b65
local               ucp-swarm-kv-certs
local               b334d329d5b66e3f7898a2d18d090f1713e3fdb4aedf1808a158f1ddb6bc902a
local               7777aafc838c6021d97b35147e6a653db68af961d0a9eb780f87bf5f072f5067
local               c18de76321643a1c2e50cb574349a13fec9b6dd672a3053db0eca375a3d7724f
local               5ea0a3ec97dd606be480ad5e868ca4352062bf85cdd82f54456106e13bf90b01
local               839e60853898c144c3acaf8600a6742846f0de4a490c9319a5b770637e9e555e
local               ce7ee8602ffab4497f91f439e49aae4b5640b424df3d187e686cc06c88518a3d
local               ucp-swarm-controller-certs
local               ucp-swarm-root-ca
local               ucp-root-ca
local               863970c15a81ef4de5aad1b7ff6bdb50591abbb7423e555776650960329e2d5f
local               2a9aa5151ef30e504b1af383e141661a7afa6b8e176f1f32d946f1378123576e
local               dec819b8961cc17c4c3963d11d8ea23b2343ba6751b497237c02345520be0c6e
local               f827c61945d1ee154aaa4de1d76bba673933232116cf5a17504115f255cc7f6d
local               7558fe809c93f860c302b0b8ecc3156b87839c582929f441a5a6cc42463e053d
local               ucp-swarm-node-certs
local               ucp-kv
local               7ca4885402efab1e62d9cdaeb098363c9c58cd999f83a27161118bee2f4d8e53
local               ucp-server-certs
local               9e2a6cfa62b2db3b1a129cff4cb458160c39a21a4fdac251134344b0bb324866
local               ucp-swarm-kv-certs
local               d861ec0504c27b49a36c28d06d95d35a825eefcf6190e84e4e98cab0103e0822
local               ucp-swarm-controller-certs
```

**Comment by vieux** [comment]
@mbentley oh my bad I kept reading "network" instead of volumes. will fix for 1.1.0
